### ISSUE TITLE
`/var/jenkins_home/.plastic4` folder has incorrect permissions in jenkins controller

### ISSUE THREAD

**Issue by Kalmalyzer** [falldamagestudio/UE-Jenkins-BuildSystem#46]
It used to have
```
drwxrwsr-x 2 root jenkins 4096 Aug  6 08:31 .plastic4
```

but recently it has
```
drwxr-sr-x 2 root jenkins 4096 Aug  6 08:31 .plastic4
```

... and this prevents Jenkins (which runs as user `jenkins`) from creating extra files in that folder. therefore the Plastic's plugin of `cm` fails when it tries to fetch Jenkinsfiles from plastic repos.

Perhaps the jenkins-controller image should include a .plastic4 folder with appropriate permissions to avoid this?

**Comment by Kalmalyzer** [comment]
Let's create the `/var/jenkins_home/.plastic4` folder manually. Fixed in c29dc57.

**Comment by Kalmalyzer** [comment]
This happened to me again, when I set up a private build system, despite the fix in c29dc57.
